openshift/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM registry.centos.org/centos/centos:7
+
+LABEL maintainer="thrcka@redhat.com"
+
+ENV LANG=en_US.UTF-8
+
+RUN yum --setopt=tsflags=nodocs install -y epel-release centos-release-openshift-origin && \
+    yum --setopt=tsflags=nodocs install -y https://download.postgresql.org/pub/repos/yum/9.4/redhat/rhel-7-x86_64/pgdg-centos94-9.4-3.noarch.rpm &&\
+    yum --setopt=tsflags=nodocs install -y postgresql awscli awk pwgen origin-clients &&\
+    yum clean all
+
+RUN mkdir -p /home/deployer/
+
+COPY env.sh deploy.sh secrets-template.yaml config-template.yaml /home/deployer/
+
+WORKDIR /home/deployer
+
+ENTRYPOINT ["./deploy.sh"]
```

Comment:
Why did you change `CMD` to `ENTRYPOINT` ?
I'd like to debug what's happening in the container and with ENTRYPOINT I can't (AFAIK) run `docker run -ti deploy-image bash`, because it always runs `deploy.sh`.
Could you change it back to `CMD` ?